provision.sh

```diff
@@ -10,7 +10,7 @@ sudo mv /tmp/tools/* /usr/local/bin
 # Remove the dnf-automatic package
 sudo dnf remove -y dnf-automatic
 
-sudo dnf install -y libvirt libvirt-devel libvirt-client git libvirt-daemon-kvm bind-utils jq gcc-c++ golang
+sudo dnf install -y libvirt libvirt-devel libvirt-client git libvirt-daemon-kvm bind-utils jq gcc-c++ golang make tmux unzip podman rsync libguestfs-tools-c zstd httpd-tools patch
```

Comment:
Feels weird to had what looks like an arbitrary list of packages to this image. Can you at least install them separately, with a comment explaining these are used by crc/snc? Would be good to understand/fix the root cause though, which is this GCP networking issue. This should not be happening :)